src/library_glfw.js

```diff
@@ -1240,7 +1274,7 @@ var LibraryGLFW = {
   glfwGetMonitors: function(count) {
     {{{ makeSetValue('count', '0', '1', 'i32') }}};
     if (!GLFW.monitors) {
-      GLFW.monitors = {{{ makeMalloc('glfwGetMonitors', POINTER_SIZE) }}};
+      GLFW.monitors = {{{ makeMalloc('glfwGetMonitors', Runtime.POINTER_SIZE) }}};
```

Comment:
The `Runtime.` here was recently removed.. perhaps this is some kind of bad merge?

Reply:
ah probably. i will undo it.